client/my-sites/email/email-management/page.jsx

```diff
@@ -0,0 +1,18 @@
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import EmailManagementHome from 'calypso/my-sites/email/email-management/email-home';
+import { emailManagement } from 'calypso/my-sites/email/paths';
+
+const EmailManagementHomePage = ( { source, selectedDomainName } ) => {
+	return (
+		<>
+			<PageViewTracker
+				path={ emailManagement( ':site', selectedDomainName ? ':domain' : null ) }
+				title="Email Home"
+				properties={ { source } }
+			/>
+			<EmailManagementHome source={ source } selectedDomainName={ selectedDomainName } />
+		</>
+	);
+};
+
+export default EmailManagementHomePage;
```

Comment:
I like this idea of suffixing page components with `Page` a lot 👍.

Reply:
However, could we make this new component a TypeScript component?

Reply:
Agreed on both points!

Reply:
Sure